.github/workflows/stacks-blockchain.yml

```diff
@@ -139,23 +139,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Determine build platforms
+        run: |
+          if [[ -z ${TAG} ]]; then
+              echo "BUILD_PLATFORMS=linux/amd64" >> $GITHUB_ENV
+          else
+              echo "BUILD_PLATFORMS=linux/amd64,linux/arm64" >> $GITHUB_ENV
+          fi
+        env:
+          TAG: ${{ github.event.inputs.tag }}
+
       - name: Set Vars
         run: |
           echo "GITHUB_SHA_SHORT=${GITHUB_SHA::7}" >> $GITHUB_ENV
           echo "GITHUB_REF_SHORT=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
 
-      - name: Build/Tag/Push Image
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            blockstack/${{ github.workflow }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            ${{ github.event.inputs.tag }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
         with:
-          repository: blockstack/${{ github.workflow }}
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-          tags: ${{ github.event.inputs.tag }}
-          tag_with_ref: true
-          add_git_labels: true
-          build_args: STACKS_NODE_VERSION=${{ github.event.inputs.tag || env.GITHUB_SHA_SHORT }},GIT_BRANCH=${{ env.GITHUB_REF_SHORT }},GIT_COMMIT=${{ env.GITHUB_SHA_SHORT }}
+
+      - name: Build/Tag/Push Image
+        uses: docker/build-push-action@v2
+        with:
+          # platforms: ${{ env.BUILD_PLATFORMS }}
+          platforms: linux/amd64,linux/arm64 # (TEMP) always build both platforms
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            STACKS_NODE_VERSION=${{ github.event.inputs.tag || env.GITHUB_SHA_SHORT }}
+            GIT_BRANCH=${{ env.GITHUB_REF_SHORT }}
+            GIT_COMMIT=${{ env.GITHUB_SHA_SHORT }}
           # Only push if (a tag was passed in) or (we're building a non-master branch which isn't a PR)
-          push: ${{ github.event.inputs.tag != '' || (github.ref != 'refs/heads/master' && !contains(github.ref, 'refs/pull')) }}
+          # push: ${{ github.event.inputs.tag != '' || (github.ref != 'refs/heads/master' && !contains(github.ref, 'refs/pull')) }}
+          push: true # (TEMP) always push this branch
```

Comment:
@zone117x Before merging, let's just make sure this and the other get reverted :)

Reply:
@zone117x Is this ready to be reverted?